client/src/components/ChannelInfo/index.js

```diff
@@ -1,12 +1,28 @@
 import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
-import { Tabs, Tab, Typography, Box } from "@material-ui/core";
+import {
+  Tabs,
+  Tab,
+  Typography,
+  Box,
+  Grid,
+  Button,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  Slide,
+} from "@material-ui/core";
 import Cookies from "js-cookie";
 import axios from "axios";
-import { githubApiDomain } from "../../utils/constants";
+import { githubApiDomain, rcApiDomain } from "../../utils/constants";
+import online from "../../images/online.png";
```

Comment:
Please move `online png` file to public and just use `/online.png` in the src of `<img>`